aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -45,7 +45,7 @@ using int_same_size_t = typename int_of_size<sizeof(T)>::type;
 template <class T>
 struct Vec256 {
 private:
-  T values[32 / sizeof(T)] = {0};
+  T values[32 / sizeof(T)];
```

Comment:
This is needed for a successful build on gcc < 4.9. Please, see this [PR](https://github.com/pytorch/pytorch/pull/22558) where the problem is discussed.

Reply:
I don't see that on the issue, I see a discussion about `{0}` and {}`.

Reply:
{0} and {} fail locally for me, but if i remove it completely i have a good build and everything seems to be working as it is (CI + tests)

Reply:
I tried `{0}` and it works for me locally (and we have gcc 4.8 tests, so it would fail there if it were a problem).

Reply:
ah, I see, this is only with the changes.  The easiest fix is probably to just do member initialization in each constructor.